app/resources/inventory.py

```diff
@@ -6,6 +6,21 @@
 from vars import game_info
 
 
+@m.user_endpoint(path=["inventory", "trade"], requires=trade_requirements)
+def trade(data: dict, user: str) -> dict:
+    element: Optional[Inventory] = wrapper.session.query(Inventory).get(data["element_uuid"])
+    target: Optional[str] = data["target"]
```

Comment:
this is not an optional parameter
```suggestion
    target: str = data["target"]
```